src/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  margin: unset;
+}
+
+.stars {
+  display: flex;
+}
+
+div .stars__star {
```

Comment:
Необов'язкова специфічність
```suggestion
.stars__star {
```

Reply:
done